modules/dsfhub-azure-cloud-account/README.md

```diff
@@ -0,0 +1 @@
+# dsfhub-azure-cloud-account
```

Comment:
Just approved the provider PR, shall we rerun the github-action to generate these docs once that's all in and pushed to the registry?

Reply:
Yep, you got it.